concepticondata/conceptlists/Lionnet-1985-364.tsv

```diff
@@ -0,0 +1,365 @@
+Lionnet-1985-364-236	236	morirse (suj sg.)	die (one)	1494	DIE
+Lionnet-1985-364-237	237	mosca	fly	1504	FLY (INSECT)
+Lionnet-1985-364-238	238	mucho	many	1198	MANY
+Lionnet-1985-364-239	239	murciélago	bat	1793	BAT
+Lionnet-1985-364-240	240	muslo	thigh	471	UPPER LEG (THIGH)
+Lionnet-1985-364-241	241	nariz	nose	1221	NOSE
+Lionnet-1985-364-242	242	nervio	nerve	2626	NERVE
+Lionnet-1985-364-243	243	nido	nest	539	NEST
+Lionnet-1985-364-244	244	nieve	snow	784	SNOW
+Lionnet-1985-364-245	245	noche, obscuridad	night, darkness	1233	NIGHT
+Lionnet-1985-364-246	246	nombre	name	1405	NAME
+Lionnet-1985-364-247	247	nube	cloud	1489	CLOUD
+Lionnet-1985-364-248	248	nuevo	new	1231	NEW
+Lionnet-1985-364-249	249	obscurecerse	darken		
+Lionnet-1985-364-250	250	ocote	pine		
+Lionnet-1985-364-251	251	ojo	eye	1248	EYE
+Lionnet-1985-364-252	252	oler (mal)	stink	1587	SMELL (STINK)
+Lionnet-1985-364-253	253	olote	degrained corncob		
+Lionnet-1985-364-254	254	ombligo	navel	1838	NAVEL
+Lionnet-1985-364-255	255	oreja	ear	1247	EAR
+Lionnet-1985-364-256	256	orinar	urinate	592	PISS
+Lionnet-1985-364-257	257	orines	urine	1142	URINE
+Lionnet-1985-364-258	258	oscuro	dark	706	DARK
+Lionnet-1985-364-259	259	pájaro	bird	937	BIRD
+Lionnet-1985-364-260	260	palma	palm	1183	PALM OF HAND
+Lionnet-1985-364-261	261	palo, madera	stick, wood	1295	STICK
+Lionnet-1985-364-262	262	panza	belly	1251	BELLY
+Lionnet-1985-364-263	263	papa	potato	593	POTATO
+Lionnet-1985-364-264	264	pararse	stand up	570	STAND UP
+Lionnet-1985-364-265	265	párpados	eyelid	17	EYELID
+Lionnet-1985-364-266	266	patear	kick	1135	KICK
+Lionnet-1985-364-267	267	pecho	chest	1592	CHEST
+Lionnet-1985-364-268	268	pedir	ask, request	1828	ASK (REQUEST)
+Lionnet-1985-364-269	269	peinarse	to comb	2070	COMB (VERB)
+Lionnet-1985-364-270	270	pelo, lana	hair, wool	2381	FEATHER OR FUR OR HAIR
+Lionnet-1985-364-271	271	pene	penis	1222	PENIS
+Lionnet-1985-364-272	272	perder en el juego	lose in a game	1660	LOSE
+Lionnet-1985-364-273	273	perro	dog	2009	DOG
+Lionnet-1985-364-274	274	perseguir	pursue	1047	PURSUE
+Lionnet-1985-364-275	275	pesado	heavy	1210	HEAVY
+Lionnet-1985-364-276	276	pescado	fish	227	FISH
+Lionnet-1985-364-277	277	picar	sting	571	STING
+Lionnet-1985-364-278	278	pie	foot	1301	FOOT
+Lionnet-1985-364-279	279	piedra	stone	857	STONE
+Lionnet-1985-364-280	280	piel	skin	763	SKIN
+Lionnet-1985-364-281	281	pierna, rodilla	leg, knee	1297	LEG
+Lionnet-1985-364-282	282	pino	pine	1740	PINE
+Lionnet-1985-364-283	283	piojo de la cabeza	head louse	310	HEAD LOUSE
+Lionnet-1985-364-284	284	pisar	tread	575	TREAD
+Lionnet-1985-364-285	285	pluma	feather	1201	FEATHER
+Lionnet-1985-364-286	286	poder	be able to	972	BE ABLE
+Lionnet-1985-364-287	287	podrirse, podre	rot	2907	ROT
+Lionnet-1985-364-288	288	prender lumbre	make fire	737	LIGHT (IGNITE)
+Lionnet-1985-364-289	289	pulga	flea	232	FLEA
+Lionnet-1985-364-290	290	punta	tip	2992	TIP (OF OBJECT)
+Lionnet-1985-364-291	291	quedarse, faltar	be left	987	REMAIN
```

Comment:
How do "quedarse" and "faltar" go together? Would therefore unmap

Reply:
Maybe because you left something (or someone didn't come) and therefore is now missing. That's why I mapped it as REMAIN. I think CONCEPTICON's definition allows this reading.

Reply:
I would still unmap. The "missing"-interpretation does not really fit the previously mapped cases which mostly relate to something staying in place (e.g. "stay", "left over", "permanecer"). As I was taught after many misleading proposals myself, the definition is not the most relevant criteria, but rather the previous mappings.

Reply:
I suggest to unmap this.

Reply:
Thanks! Now is more clear to me the workflow in this type of cases :)